sample.py

```diff
@@ -0,0 +1,4 @@
+print('hello world')
```

Comment:
This line is useless. Please remove it and merge this branch. Also delete the branch

Reply:
okay I'll do it. :)

Reply:
I have made necessary changes. Please review again.